android-components/plugins/dependencies/src/main/java/DependenciesPlugin.kt

```diff
@@ -53,7 +53,7 @@ object Versions {
 
     // see https://android-developers.googleblog.com/2022/06/independent-versioning-of-Jetpack-Compose-libraries.html
     // for Jetpack Compose libraries versioning
-    const val compose_version = "1.4.3"
+    const val compose_version = "1.5.0-alpha01"
```

Comment:
Please do not ship alpha/beta versions of dependencies.

Reply:
Done, reverted